<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Light</class>
 <widget class="QDialog" name="Light">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1874</width>
    <height>937</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>30</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pushButton_image">
   <property name="geometry">
    <rect>
     <x>1050</x>
     <y>350</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pButton_work">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>630</y>
     <width>470</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(186, 189, 182);</string>
   </property>
   <property name="text">
    <string>舱门到工作位</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pButton_hard">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>630</y>
     <width>501</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(186, 189, 182);</string>
   </property>
   <property name="text">
    <string>舱门回硬件位</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_conut">
   <property name="geometry">
    <rect>
     <x>1050</x>
     <y>60</y>
     <width>441</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="text">
    <string>计算区域：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_x">
   <property name="geometry">
    <rect>
     <x>1050</x>
     <y>170</y>
     <width>201</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="text">
    <string>center  X：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_radius">
   <property name="geometry">
    <rect>
     <x>1050</x>
     <y>260</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Radius：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_y">
   <property name="geometry">
    <rect>
     <x>1420</x>
     <y>160</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Y：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lEdit_X">
   <property name="geometry">
    <rect>
     <x>1260</x>
     <y>164</y>
     <width>121</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="lEdit_rad">
   <property name="geometry">
    <rect>
     <x>1220</x>
     <y>250</y>
     <width>113</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="lEdit_Y">
   <property name="geometry">
    <rect>
     <x>1480</x>
     <y>160</y>
     <width>113</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label_light">
   <property name="geometry">
    <rect>
     <x>1150</x>
     <y>420</y>
     <width>221</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="text">
    <string>荧光亮度:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_light_num">
   <property name="geometry">
    <rect>
     <x>1380</x>
     <y>420</y>
     <width>151</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>31</pointsize>
    </font>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cBox_panel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>700</y>
     <width>551</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::DefaultContextMenu</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">selection-color: rgb(186, 189, 182);
color: rgb(0, 0, 0);
background-color: rgb(239, 239, 239);</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pButton_set">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>700</y>
     <width>171</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(186, 189, 182);</string>
   </property>
   <property name="text">
    <string>设置</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pButton_get_light">
   <property name="geometry">
    <rect>
     <x>1150</x>
     <y>500</y>
     <width>301</width>
     <height>111</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/images/confirmbt.png);
</string>
   </property>
   <property name="text">
    <string>   获取亮度</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pButton_backl">
   <property name="geometry">
    <rect>
     <x>1550</x>
     <y>730</y>
     <width>301</width>
     <height>111</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/images/back.png);</string>
   </property>
   <property name="text">
    <string>   返回</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pButton_set_xy">
   <property name="geometry">
    <rect>
     <x>1410</x>
     <y>250</y>
     <width>171</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(186, 189, 182);</string>
   </property>
   <property name="text">
    <string>设置</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>951</width>
     <height>591</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(160, 250, 250);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_machine">
   <property name="geometry">
    <rect>
     <x>1420</x>
     <y>0</y>
     <width>171</width>
     <height>71</height>
    </rect>
   </property>
   <property name="currentText">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
